.circleci/config.yml

```diff
@@ -483,7 +487,7 @@ jobs:
             - run:
                 name: Decode Apple Certificates
                 context: org-global
-                command: bash -c '[ -z $APPLE_CERTIFICATES ] || base64 -D -o frontend/wallet/Certificates.p12 <<< $APPLE_CERTIFICATES'
+                command: bash -c '[ -z $APPLE_CERTIFICATES ] || base64 -D -o frontend/wallet/Certificates.p12 \<<< $APPLE_CERTIFICATES'
```

Comment:
is the `\` intentional here?

Reply:
Yea, the 2.1 version of CircleCI's config uses `<<` and `>>` to delimit pipeline variables (e.g. the one added in this PR), so they need escaping when we want the normal version.